browser/index.html

```diff
@@ -12,6 +12,7 @@
         https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Viewport_basics
     -->
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="/dasher-web/browser/favicon.ico"/>
```

Comment:
Hi @gavinhenderson , Can this be a relative path instead? I guess it would be just `href="favicon.ico"`